src/topology/continuous_function/algebra.lean

```diff
@@ -639,6 +657,18 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [topological_space R] {M : Type*} [topological_space M]
+  [has_scalar R M] [has_continuous_smul R M] :
+  has_continuous_smul R C(α, M) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  have h : continuous (λ x : (R × C(α, M)) × α, x.fst.snd x.snd) :=
+    continuous_ev.comp ((continuous_snd.comp continuous_fst).prod_mk continuous_snd),
```

Comment:
You'll need to replace `continuous_ev` with `continuous_eval'` throughout once #12738 finishes merging.